easybuild/easyconfigs/i/IGV/IGV-2.5.0-Java-11.eb

```diff
@@ -0,0 +1,35 @@
+# EasyBuild easyconfig
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'IGV'
+version = '2.5.0'
+versionsuffix = '-Java-11'
+
+homepage = 'http://www.broadinstitute.org/software/igv/'
+description = """This package contains command line utilities for
+ preprocessing, computing feature count density (coverage),  sorting, and
+ indexing data files."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/' +
+               '%(version_major)s.%(version_minor)s']
```

Comment:
This can just be a single line (it's well below the 120 chars limit)?

```python
source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/%(version_major)s.%(version_minor)s']